client/dashboard/components/page-view-tracker/index.tsx

```diff
@@ -0,0 +1,17 @@
+import { useMatch } from '@tanstack/react-router';
+import { useEffect } from 'react';
+import { useAnalytics } from '../../app/analytics';
+
+export function PageViewTracker( { title }: { title: string } ) {
+	const { recordPageView } = useAnalytics();
+	const path = useMatch( {
+		strict: false,
+		select: ( match ): string => match.context.config.basePath + match.routeId,
```

Comment:
Unlike Calypso's `<PageViewTracker>` we're able to access the route pattern (i.e. the route without the real site slug in it) so page's don't have to repeat it.